src/components/Icon/Icon.tsx

```diff
@@ -28,7 +28,7 @@ const withStylesWrapper =
       </SvgWrapper>
     );
 
-const SvgWrapper = styled.div<Partial<IconProps>>`
+const SvgWrapper = styled.span<Partial<IconProps>>`
```

Comment:
The `div` on the icon wrapper was causing nesting warnings. ie. `div can't be inside a p`